src/main/java/hudson/plugins/git/extensions/impl/PruneStaleTag.java

```diff
@@ -0,0 +1,120 @@
+/*
+ * The MIT License
+ *
+ * Copyright (c) 2017 CloudBees, Inc.
```

Comment:
Copyright should be 2020 or 2019 (whenever the file was created).
```suggestion
 * Copyright (c) 2020 Nikolas Falco
```